frontend/src/components/taxons/TaxonsRoute.tsx

```diff
@@ -1,15 +1,26 @@
 import React from 'react'
 import { useAppSelector } from '../../hooks'
 import { selectTaxonsByID } from '../../selectors'
-import { getAllItems } from '../../models/frontend_models'
+import { Taxon, getAllItems } from '../../models/frontend_models'
 import TaxonsListContent from './TaxonsListContent'
+import PageContainer from '../PageContainer'
+import { Navigate, Route, Routes } from 'react-router-dom'
+import { AddTaxonRoute, EditTaxonRoute } from './EditTaxon'
 
-function TaxonsRoute() {
+const TaxonsRoute = () => {
 	const taxonsState = useAppSelector(selectTaxonsByID)
-	const taxons = getAllItems(taxonsState)
+	const taxons: Taxon[] = getAllItems(taxonsState)
```

Comment:
`taxons` and `taxonsState` are not being used, so these two lines can be removed.